setup.py

```diff
@@ -62,12 +62,17 @@
         "llvmlite<=0.34.0;python_version>='3.6'",
         "llvmlite<=0.31.0;python_version<'3.6'",
     ],
+    "opentelemetry": [
+        "opentelemetry-api==0.9b0",
+        "opentelemetry-sdk==0.9b0",
+        "opentelemetry-instrumentation==0.9b0 ",
+    ],
 }
 
 all_extras = []
 
 for extra in extras:
-    if extra == "fastparquet":
+    if extra == "fastparquet" or extra == "opentelemetry":
         # Skip fastparquet from "all" because it is redundant with pyarrow and
         # creates a dependency on pre-release versions of numpy. See:
         # https://github.com/googleapis/google-cloud-python/issues/8549
```

Comment:
Let's document why we're excluding `opentelemetry` so that it can be added back to `all` after this package drops Python 2 support.

```suggestion
    if extra in (
        # fastparquet is redundant with pyarrow and creates a dependency
        # on pre-release versions of numpy. See:
        # https://github.com/googleapis/google-cloud-python/issues/8549
        "fastparquet",
        # opentelemtry is not compatible with Python 2.
        "opentelemetry",
    ):
```